static/app/components/events/breadcrumbs/utils.tsx

```diff
@@ -0,0 +1,143 @@
+import styled from '@emotion/styled';
+
+import type {SelectOption} from 'sentry/components/compactSelect';
+import type * as Timeline from 'sentry/components/timeline';
+import {
+  IconCursorArrow,
+  IconFire,
+  IconFix,
+  IconInfo,
+  IconLocation,
+  IconMobile,
+  IconRefresh,
+  IconSort,
+  IconSpan,
+  IconStack,
+  IconTerminal,
+  IconUser,
+  IconWarning,
+} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {BreadcrumbType, type RawCrumb} from 'sentry/types/breadcrumbs';
+import {toTitleCase} from 'sentry/utils/string/toTitleCase';
+
+export const BREADCRUMB_TIMESTAMP_PLACEHOLDER = '--';
+const BREADCRUMB_TITLE_PLACEHOLDER = t('Generic');
+
+export enum BreadcrumbTimeDisplay {
```

Comment:
just encouraging people to use const enum where possible
```suggestion
export const enum BreadcrumbTimeDisplay {
```